src/config/entities/__tests__/configuration.ts

```diff
@@ -55,6 +63,15 @@ export default (): ReturnType<typeof configuration> => ({
           56: { chainName: faker.string.sample() },
           8453: { chainName: faker.string.sample() },
         },
+        currencies: [
+          ...Array.from({ length: faker.number.int({ min: 2, max: 5 }) }, () =>
+            faker.finance.currencyCode().toLowerCase(),
+          ),
+          'btc',
+          'eth',
+          'eur',
+          'usd',
+        ],
```

Comment:
As above regarding duplicates.

Reply:
Also done in [d6f35a0](https://github.com/safe-global/safe-client-gateway/pull/1099/commits/d6f35a006240b85a4e1f2a9743a33342c0b1e1c4)